inc/class.client.php

```diff
@@ -288,6 +288,31 @@ public static function get_rel_attribut() {
 		return $rel;
 	}
 
+	/**
+	 * Retrieve the post count for a term across all taxonomies.
+	 * 
+	 * This function queries terms based on the given term name and returns the number of posts associated with the first matching term.
+	 * It does not require specifying a taxonomy and will search across all public taxonomies.
+	 * 
+	 * @param string $item
+	 * @return int The number of posts associated with the first matching term. Returns 0 if no term is found or if it has no posts.
+	 */
+	public static function get_term_post_counts( $item ) {
+
+			$terms = get_terms( array(
+				'name' => strip_tags($item),
```

Comment:
Hi @Deji98 Add 'number' => 1 to the query

Reply:
Okay, done that right now @ojopaul